packages/hx-library/src/components/hx-carousel/hx-carousel.test.ts

```
@@ -1268,6 +1283,119 @@ describe('hx-carousel', () => {
+      await el.updateComplete;
+      expect(el['_currentIndex']).toBe(2);
+      expect(shadowQuery<HTMLElement>(el, '.live-region')?.textContent?.trim()).toBe('Slide 3 of 3');
+
+      const events: Array<{ index: number; slide: unknown }> = [];
+      el.addEventListener('hx-slide-change', (e) => {
+        const d = (e as CustomEvent<{ index: number; slide: unknown }>).detail;
+        events.push({ index: d.index, slide: d.slide });
+      });
+
+      // Remove every slide at runtime.
+      el.querySelectorAll('hx-carousel-item').forEach((s) => s.remove());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)